README.md

```diff
@@ -79,7 +79,7 @@ Compiling:
 cd FCCDetectors
 mkdir build
 cd build
-cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DCMAKE_BUILD_TYPE=RELWITHDEBINFO -DCMAKE_CXX_STANDARD=17
+cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DWITH_DD4HEP=ON
```

Comment:
Why are you proposing these changes in the README ?

Reply:
When compiling FCCDetectors, k4SimGeant4 and k4RecCalorimeter using the original cmake line:
cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DCMAKE_BUILD_TYPE=RELWITHDEBINFO -DCMAKE_CXX_STANDARD=17
there will be an error like:
g++: error: unrecognized command-line option '-std=c++17-O2'; did you mean '-std=c++17'?

Reply:
Ok